processing/src/main/java/io/druid/segment/data/BlockLayoutSerde.java

```diff
@@ -0,0 +1,309 @@
+package io.druid.segment.data;
+
+
+import com.google.common.base.Supplier;
+import com.google.common.io.ByteSink;
+import com.google.common.io.ByteStreams;
+import com.google.common.io.Closeables;
+import com.google.common.io.CountingOutputStream;
+import com.google.common.primitives.Ints;
+import com.metamx.common.guava.CloseQuietly;
+import io.druid.collections.ResourceHolder;
+import io.druid.collections.StupidResourceHolder;
+import io.druid.segment.CompressedPools;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.nio.LongBuffer;
+import java.nio.channels.Channels;
+import java.nio.channels.ReadableByteChannel;
+import java.nio.channels.WritableByteChannel;
+
+public class BlockLayoutSerde
```

Comment:
Github is kind of hard to read. Is there anything actually in this class? if it is just a wrapper for the other classes can you just make it a package instead?